rts/builds/headless/CMakeLists.txt

```diff
@@ -15,6 +15,24 @@ set(OpenGL_GL_PREFERENCE LEGACY)
 
 find_package(OpenGL 3.0 REQUIRED)
 
+# NOTE(create_headless_target): We don't need to copy COMPILE_DEFINTIONS till the add_definitions are used.
```

Comment:
This note is a bit unclear to me... do you mean something like:

> We don't need to copy COMPILE_DEFINTIONS because the headless defines are defined at the folder level and we don't want to inherit original ones

?

Reply:
Current target flow is: 
Orginal Game target in spring legacy -> target copy in headless

`add_definitions` works at current CMakeList.txt and it's sub-directories (which makes reasoning about correct definitions in singular target hard to grasp). 

When we're making copy of the target we want to avoid copies of `COMPILE_DEFINITIONS` because these will be from spring-legacy and we want different set for headless. The define set will be inhered by the place where the target is defined (in headless cmake file).
I agree that comment isn't descriptive enough.